README.md

```diff
@@ -38,91 +36,135 @@
 
-## Guiding Principles
+- homogeneous (with nodes and links of one type),
+- heterogeneous (with more than one type of nodes and/or links)
+- knowledge graphs (extreme heterogeneous graphs with thousands of types of edges)
+- graphs with or without data associated with nodes
+- graphs with edge weights
 
-StellarGraph uses the [Keras](https://keras.io/) API as implemented in the [TensorFlow](https://tensorflow.org/) library and adheres to the same
-guiding principles as Keras: user-friendliness, modularity, and easy extendability. Modules and layers
-of StellarGraph library are designed so that they can be used together with
-standard Keras layers and modules, if required. This enables flexibility in using existing
-or creating new models and workflows for machine learning on graphs.
+StellarGraph is built on [TensorFlow 2](https://tensorflow.org/) and its [Keras high-level API](https://www.tensorflow.org/guide/keras), as well as [Pandas](https://pandas.pydata.org) and [NumPy](https://www.numpy.org). It is thus user-friendly, modular and extensible. It interoperates smoothly with code that builds on these, such as the standard Keras layers and [scikit-learn](http://scikit-learn.github.io/stable), so it is easy to augment the core graph machine learning algorithms provided by StellarGraph.
 
 ## Getting Started
 
-To get started with StellarGraph you'll need data structured as a homogeneous or heterogeneous graph, including
-attributes for the entities represented as graph nodes.
-[NetworkX](https://networkx.github.io/) is used to represent the graph and [Pandas](https://pandas.pydata.org/)
-or [Numpy](https://www.numpy.org/) are used to store node attributes.
+[The numerous detailed and narrated examples](https://github.com/stellargraph/stellargraph/tree/master/demos/) are a good way to get started with StellarGraph. There is likely to be one that is similar to your data or your problem (if not, [let us know](#getting-help)).
+
+To run these examples, you'll need to install StellarGraph. One of the following might be appropriate:
+
+- Using pip: `pip install stellargraph[demos]`
+- Using conda: `conda install -c stellargraph stellargraph`
+
+(See [Installation](#Installation) section for more details and more options.)
+
+## Getting Help
+
+If you get stuck or have a problem, there's many ways to make progress and get help or support:
+
+- [Read the documentation](https://stellargraph.readthedocs.io)
+- [Consult the examples](https://github.com/stellargraph/stellargraph/tree/master/demos/)
+- Contact us:
+  - [Ask questions and discuss problems on the StellarGraph Discourse forum](https://community.stellargraph.io)
+  - [File an issue](https://github.com/stellargraph/stellargraph/issues/new/choose)
+
+
+## Example: GCN
```

Comment:
I think the GCN example is unnecessary here.

Reply:
Could you expand on why you think so?

I added the example because it's widely recommended (e.g. https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3 https://www.makeareadme.com) and it lets people get a sense for the library and how it uses Pandas and Keras and so on at a glance, before they're committed and digging through the demos.

Reply:
It looks abrupt to me. The readme is meant for a general introduction. An example, that also, a specific one looks out of place. Pointing to the detailed GCN demo, as you did later, in my opinion would have been enough.  Also, from the link you refer above, it says to give a concise example. The GCN is this form would confuse and turn me away more than making it simpler for me, or I would just skip this section, but then again it could be just me, go with the widely recommended choice.

Reply:
I don't think pointing to the GCN demo is quite enough: at least at the moment, https://github.com/stellargraph/stellargraph/blob/master/demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb is too specific to serve as a good high-level example, focusing on the "big picture" view, since it has to explain every step of the process. My goal with this example is to give that big picture view of how StellarGraph fits into the eco-system, in particular emphasising that connects smoothly to Pandas and TensorFlow/Keras, rather than being a fundamentally new ML paradigm to learn.

It's definitely true that we should be concerned about it being confusing and intimidating, so I've restructured it to interleave real headings and text with the code. Do you think this is better?

Reply:
The example looks more readable! Nice! 
Added some further comments incase you find them helpful.